src/main/java/frc/robot/CommandSwerveDrivetrain.java

```diff
@@ -36,6 +44,16 @@ public CommandSwerveDrivetrain(
     if (Utils.isSimulation()) {
       startSimThread();
     }
+
+    SmartDashboard.putData("Field", field);
+  }
+
+  public BiConsumer<Pose2d, Double> getVisionMeasurementConsumer() {
+    return (t, u) -> addVisionMeasurement(t, u);
+  }
```

Comment:
Keep this, remove the other changes to this file

Reply:
Why? Does it conflict with changes made on main, or is there some other reason?